src/main/java/org/elasticsearch/index/query/IdsQueryBuilder.java

```diff
@@ -140,7 +130,7 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
             builder.value(value);
         }
         builder.endArray();
-        if (boost != -1) {
+        if (boost != 1.0f) {
```

Comment:
can you double check @cbuescher that this was a bug introduced while migrating this query?